server/chat-commands/info.js

```diff
@@ -2339,6 +2339,18 @@ const commands = {
 	},
 	pickrandomhelp: [`/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements.`],
 
+	'!shuffle': true,
+	shuffle(target, room, user) {
+		if (!target) return false;
+		if (!target.includes(',')) return this.parse('/help shuffle');
+		if (!this.runBroadcast(true)) return false;
+		let args = target.split(',');
+		if (args.length < 3) return this.errorReply('Give at least 3 arguments to shuffle.');
+		const results = Dex.shuffle(args);
```

Comment:
hyper nit- `Dex.shuffle` works in place, using the return value here might just be misleading.

Reply:
```suggestion
		const results = Dex.shuffle(args.map(arg => arg.trim()));
```

This is irrelevant because the args need to be trimmed anyway.